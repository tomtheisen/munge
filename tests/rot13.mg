(
    /[a-m]/i => {_ ord 13 + chr}
    /[n-z]/i => {_ ord 13 - chr}
)