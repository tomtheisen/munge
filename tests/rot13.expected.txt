 Guvf Fbhepr Pbqr Sbez vf fhowrpg gb gur grezf bs gur Zbmvyyn Choyvp
   - Yvprafr, i. 2.0. Vs n pbcl bs gur ZCY jnf abg qvfgevohgrq jvgu guvf svyr,
   - Lbh pna bognva bar ng uggc://zbmvyyn.bet/ZCY/2.0/. 